### PR TITLE
LG/DCOS-54120: Modify the replace a master node topic for 1.11, 1.12,…

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
… and 1.13

## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.
https://jira.mesosphere.com/browse/DCOS-54120

## Description of changes being made
Slack request:
Joe Thompson [11:24 AM]
Hey docs folks, Support is currently helping GE-T through an issue they had because they replaced masters too fast.  Can we get a big red "WARNING: you have to do this part" box at the top of this step in the "Replace a master node" doc?  https://docs.mesosphere.com/1.11/administering-clusters/replacing-a-master-node/#check-that-the-new-master-is-healthy
also it might be good to have users take a ZK/CockroachDB backup at the beginning, before shutting down the master to replace

## Checklist
- [X] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/1.11/administering-clusters/replacing-a-master-node/index.md [hunk 1]
```diff
@@ -4,43 +4,44 @@ navigationTitle:  Replace a master node
...
 excerpt: Replacing a master node in an existing DC/OS cluster
-
 enterprise: true
 ---
-
 <!-- This source repo for this topic is https://github.com/mesosphere/dcos-docs-site -->
+You can replace a master node in an existing DC/OS cluster. You should keep in mind, however, that you should only ever replace one master at a time. The following steps summarize how to replace a master node for a DC/OS cluster.
 
+# To replace a master node
+1. Back up ZooKeeper state information using Exhibitor, the Guano utility, or a custom script.
```
Comment (omkensey) [inline pages/1.11/administering-clusters/replacing-a-master-node/index.md]: We should just link to https://docs.mesosphere.com/1.11/installing/installation-faq/#q-how-do-i-backup-zookeeper-using-guano (mutatis mutandis for other versions)

File: pages/1.11/administering-clusters/replacing-a-master-node/index.md [hunk 2]
```diff
@@ -4,43 +4,44 @@ navigationTitle:  Replace a master node
...
-### Add the new master
+    For example, if you download and extract the Guano utility, you can run a command similar to the following on the master node:
 
-The procedure for adding the new master node to replace the one taken offline in the previous step is quite simple.
+    ```bash
+    dcos-shell java -jar guano-0.1a.jar -u super -p secret -d / -o /tmp/mesos-zk-backup -s $ZKHOST:2181 && tar -zcvf zkstate.tar.gz /tmp/mesos-zk-backup/
+    ```
 
-#### `master_discovery: static`
+1. Back up the DC/OS identity and access management CockroachDB database to a file by running a command similar to the following on the master node:
```
Comment (omkensey) [inline pages/1.11/administering-clusters/replacing-a-master-node/index.md]: Similar to previous note, we should just link to https://docs.mesosphere.com/1.11/installing/installation-faq/#q-how-do-i-backup-the-iam-database-enterprise

File: pages/1.12/administering-clusters/replacing-a-master-node/index.md [hunk 3]
```diff
@@ -4,33 +4,43 @@ navigationTitle:  Replace a master node
...
 enterprise: true
 ---
+You can replace a master node in an existing DC/OS cluster. You should keep in mind, however, that you should only ever replace one master at a time. The following steps summarize how to replace a master node for a DC/OS cluster.
 
-<!-- This source repo for this topic is https://github.com/mesosphere/dcos-docs-site -->
-
-
-You can replace a master node in an existing DC/OS cluster. Only one master should be replaced at a time.
+# To replace a master node
+1. Back up ZooKeeper state information using Exhibitor, the Guano utility, or a custom script.
```
Comment (omkensey) [inline pages/1.12/administering-clusters/replacing-a-master-node/index.md]: See 1.11 note

File: pages/1.12/administering-clusters/replacing-a-master-node/index.md [hunk 4]
```diff
@@ -4,33 +4,43 @@ navigationTitle:  Replace a master node
...
-## Remove the existing master
+    For example, if you download and extract the Guano utility, you can run a command similar to the following on the master node:
 
-To begin, shut down the master node you want to replace.
+    ```bash
+    dcos-shell java -jar guano-0.1a.jar -u super -p secret -d / -o /tmp/mesos-zk-backup -s $ZKHOST:2181 && tar -zcvf zkstate.tar.gz /tmp/mesos-zk-backup/
+    ```
 
-## Add the new master
+1. Back up the DC/OS identity and access management CockroachDB database to a file by running a command similar to the following on the master node:
```
Comment (omkensey) [inline pages/1.12/administering-clusters/replacing-a-master-node/index.md]: See 1.11 note

File: pages/1.13/administering-clusters/replacing-a-master-node/index.md [hunk 5]
```diff
@@ -6,27 +6,41 @@ menuWeight: 800
 excerpt: Replacing a master node in an existing DC/OS cluster
 enterprise: true
 ---
+You can replace a master node in an existing DC/OS cluster. You should keep in mind, however, that you should only ever replace one master at a time. The following steps summarize how to replace a master node for a DC/OS cluster.
 
-You can replace a master node in an existing DC/OS cluster. Only one master should be replaced at a time.
+# To replace a master node
+1. Back up ZooKeeper state information using Exhibitor, the Guano utility, or a custom script.
```
Comment (omkensey) [inline pages/1.13/administering-clusters/replacing-a-master-node/index.md]: See 1.11 note

File: pages/1.13/administering-clusters/replacing-a-master-node/index.md [hunk 6]
```diff
@@ -6,27 +6,41 @@ menuWeight: 800
...
-## Remove the existing master
+    For example, if you download and extract the Guano utility, you can run a command similar to the following on the master node:
 
-To begin, shut down the master node you want to replace.
+    ```bash
+    dcos-shell java -jar guano-0.1a.jar -u super -p secret -d / -o /tmp/mesos-zk-backup -s $ZKHOST:2181 && tar -zcvf zkstate.tar.gz /tmp/mesos-zk-backup/
+    ```
 
-## Add the new master
+1. Back up the DC/OS identity and access management CockroachDB database to a file by running a command similar to the following on the master node:
```
Comment (omkensey) [inline pages/1.13/administering-clusters/replacing-a-master-node/index.md]: See 1.11 note

File: pages/1.11/administering-clusters/replacing-a-master-node/index.md (hunk 1 above)
Comment (jgehrcke) [reply pages/1.11/administering-clusters/replacing-a-master-node/index.md]: CC @timaa2k @adamtheturtle @jongiddy is that still the correct reference?

Comment (adamtheturtle) [reply pages/1.11/administering-clusters/replacing-a-master-node/index.md]: I'm not sure but this will be changing very soon, right @timaa2k ?

Comment (lsgunnlsgunn) [reply pages/1.11/administering-clusters/replacing-a-master-node/index.md]: If the FAQ info for the back up steps are still valid, I can link to them. I'm not completely confident in whether they are accurate or legacy info. It raises some questions (for me anyway): is dcos-shell a standalone CLI package? how/when is it installed?

Comment (timaa2k) [reply pages/1.11/administering-clusters/replacing-a-master-node/index.md]: It is the correct reference until we release a new way of backing up ZooKeeper.
At this point it is not clear whether we will backport the new way.

**Pull request merged**